scripts/testlib.sh

```diff
@@ -165,7 +165,11 @@ start-ganache() {
 	# shellcheck disable=SC2155
 	local ganache_exec="$(npm config get prefix)/bin/ganache"
 	check-set-swap-test-data-dir
-	NODE_OPTIONS="--max_old_space_size=8192" nohup \
+	local nohup_cmd=nohup
+	if [[ "$(uname)" == 'Darwin' ]]; then
+		nohup_cmd=
+	fi
```

Comment:
Not sure what was going on here.  Using nohup was causing the ganache command to fail:
```
nohup: can't detach from console: Inappropriate ioctl for device
```
I didn't try from a graphical console. All the work was done over ssh.

As a side note, you can run ganache from docker, eliminating the need to install ganache:
```
docker run --name ganache -p 8545:8545 --detach --rm trufflesuite/ganache:latest --deterministic --accounts=50 --miner.blockTime=1
```